internal/server/data/migrations_test.go

```diff
@@ -646,6 +647,8 @@ type testCaseLabel struct {
 	Line string
 }
 
+var isEnvironmentCI = os.Getenv("CI") != ""
```

Comment:
Should this variable live closer to there it is used? I dont see it in `migrations_test.go`

Reply:
It's used 6 lines below here. The diff looks weird because I removed it from a different place in the `data` package.